terraform/vars/terraform-production.tfvars

```diff
@@ -1,11 +1,11 @@
 environment             = "production"
 backup_retention_period = 7
 log_retention_period    = 30
-rds_version             = "12.8"
+rds_version             = "14.15"
 rds_instance_class      = "db.r6g.large"
```

Comment:
why do we want to keep production at db.r6.large and increase dev and staging to db.r6g.xlarge? is that intentional?